.github/workflows/push.yaml

```diff
@@ -16,4 +16,4 @@ jobs:
         with:
           name: ngi
           authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
-      - run: nix flake check
+      - run: nix flake check --debug --print-build-logs
```

Comment:
```suggestion
      - run: nix flake check --verbose --print-build-logs
```

Reply:
See https://github.com/ngi-nix/ngipkgs/pull/67#discussion_r1372089882.

Reply:
OK. In such cases consider creating a new branch on top of the branch you are trying to merge, make the failure-diagnosis related changes in a new commit, and push to the new branch. That way there'll be no noise in the PR, and once you've resolved the issue, you can commit it to the PR branch. No big deal about the change in this instance, I am just trying to share a tip here.

Also note that this won't work for Actions that run on pull request only, like the on in `.github/workflows/pull_request.yaml`. That's why I am commenting here.